### PR TITLE
Update Ecosystem.md

### PR DESCRIPTION
**Pull request by GiovanniCardamone**
Hello, i created a simple extension that allow user to map some directory to automatic handle hierarchy and map it to urls.

Currently I'm working on improve doc, but you can check it out here:
https://github.com/GiovanniCardamone/fastify-autoroutes

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Ecosystem.md [hunk 1]
```diff
@@ -148,3 +148,4 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`i18next-http-middleware`](https://github.com/i18next/i18next-http-middleware#fastify-usage) An [i18next](https://www.i18next.com) based i18n (internationalization) middleware to be used with Node.js web frameworks like express or `Fastify` and also for Deno.
 - [`k-fastify-gateway`](https://github.com/jkyberneees/fastify-gateway) API Gateway plugin for `fastify`, a low footprint implementation that uses the `fastify-reply-from` HTTP proxy library.
 - [`openapi-validator-middleware`](https://github.com/PayU/openapi-validator-middleware#fastify) Swagger and OpenAPI 3.0 spec-based request validation middleware that supports `fastify`.
+- [`fastify-autoroutes`](https://github.com/GiovanniCardamone/fastify-autoroutes) Map directory structure to routes url
```
Comment (RafaelGSS) [inline docs/Ecosystem.md:151]: The ecosystem plugins is sorted by name, could change it?

Comment (kibertoad) [reply docs/Ecosystem.md:151]: Dot is missing at the end of the sentence. Also could you make description a bit more consistent with the others? Most of them are "X is a plugin/middleware for YZ"

Comment (GiovanniCardamone) [reply docs/Ecosystem.md:151]: yes sorry, i will change it

**Comment by GiovanniCardamone** [comment]
Yes i am currently working on it!

**Comment by GiovanniCardamone** [comment]
I created some testcase and added the code coverage.

**Pull request merged**